doc/user_manual/postprocessor.tex

```diff
@@ -149,6 +149,19 @@ \subsubsection{BasicStatistics}
   quantities are going to be calculated, if \textit{False} unbiased.
   \default{False}
   %
+  \item \xmlNode{dataset}, \xmlDesc{string (boolean), optional field}, if \textit{True} \xmlString{DataSet}
+    will be used to store the calculation results, if \textit{False} \xmlString{PointSet} or \xmlString{HistorySet}
+    will be used to store the calculation results.
+    \nb The optional \xmlString{DataSet} is added only to this Post-Processor, one can still use the \xmlString{OutStreams}
+    to print the variables available in the \textit{DataSet}. The \xmlString{"metric"} names are used as the
+    variable names, i.e. variable names listed in \xmlNode{Input} or \xmlNode{Output}. If scalar metrics, such as 
+    \xmlNode{expectedValue} and \xmlNode{variance}, are requested, the index variable \xmlString{targets} will be required.
+    If vector metrics, such as \xmlNode{covariance} and \xmlNode{sensitivity}, are requested, the index variables
+    \xmlString{targets} and \xmlString{features} will be required. If \xmlNode{percentile} is requested, an additional
+    index variable \xmlString{percent} should be added. In addition, if dynamic BasicStatistics (e.g. time) is requested,
+    the index variable \xmlString{time}  will be required.
```

Comment:
you should put an example XML section...It is quite hard to follow. Maybe a bulleted list might work better.
Eg.

- If scalar metrics, such as  \xmlNode{expectedValue} and \xmlNode{variance} bla blas
- If percentile, bla bla
etc

Reply:
updated